spec/models/comment_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'validations' do
+    it 'is invalid without a body' do
+      user = User.create(name: 'John Doe')
+      post = Post.create(title: 'Lorem ipsum', comments_counter: 0, likes_counter: 0, author_id: user.id, author: user)
+      comment = Comment.new(post_id: post.id, user_id: user.id)
+      expect(comment).to_not be_valid
+    end
+  end
+
+  describe 'associations' do
+    it 'belongs to a post' do
+      association = described_class.reflect_on_association(:post)
+      expect(association.macro).to eq :belongs_to
+    end
+
+    it 'belongs to a user' do
+      association = described_class.reflect_on_association(:user)
+      expect(association.macro).to eq :belongs_to
+    end
+  end
```

Comment:
- [ ] According to the requirements, please test all your model's methods. In this case, `#update_post_counter` is missing its respective test. 🤓 🔧

Reply:
This test has been added